nvflare/private/fed/app/simulator/simulator.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Federated server launching script."""
+
+import argparse
+import os
+import sys
+
+from nvflare.private.fed.app.simulator.simulator_runner import SimulatorRunner
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("job_folder")
+    parser.add_argument("--workspace", "-o", type=str, help="WORKSPACE folder", required=True)
+    parser.add_argument("--clients", "-n", type=int, help="number of clients")
+    parser.add_argument("--client_list", "-c", type=str, help="client names list")
+    parser.add_argument("--threads", "-t", type=int, help="number of running threads", required=True)
+    parser.add_argument("--set", metavar="KEY=VALUE", nargs="*")
+    args = parser.parse_args()
+    return args
+
+
+if __name__ == "__main__":
+    """
+    This is the main program when starting the NVIDIA FLARE server process.
+    """
+
+    if sys.version_info >= (3, 9):
+        raise RuntimeError("Python versions 3.9 and above are not yet supported. Please use Python 3.8 or 3.7.")
+    if sys.version_info < (3, 7):
+        raise RuntimeError("Python versions 3.6 and below are not supported. Please use Python 3.8 or 3.7.")
+    args = parse_args()
+
+    simulator = SimulatorRunner(args)
```

Comment:
suggest change this file name to 
`simulate.py` and `SimulatorRunner` ->`Simulator`